CODE/Sources/PennyBOT/Penny.swift

```diff
@@ -52,4 +48,42 @@ struct Penny {
         
         RunLoop.current.run()
     }
+    
+    static func bootstrapLoggingSystem(httpClient: HTTPClient) {
+        guard let webhookUrl = Constants.loggingWebhookUrl,
+              let token = Constants.botToken
+        else {
+            fatalError("Missing 'LOGGING_WEBHOOK_URL' or 'BOT_TOKEN' env vars")
+        }
+        DiscordGlobalConfiguration.logManager = DiscordLogManager(
+            client: DefaultDiscordClient(
+                httpClient: httpClient,
+                token: token,
+                appId: nil
+            ),
+            configuration: .init(
+                fallbackLogger: Logger(
+                    label: "DiscordBMFallback",
+                    factory: StreamLogHandler.standardOutput(label:metadataProvider:)
+                ),
+                aliveNotice: .init(
```

Comment:
every 3 hours Penny will send a "I'm alive" notice to the logs. This is only if there has been no logs for 3 hours.
also when the bot restarts, Penny will mention me incase it's crashing or something.